### PR TITLE
Fix #74 - Add support for Django 3.0

### LINKED ISSUE #74: Add Django 3.0
We should add support for Django 3.0. We should also add Django to `classifiers` and `install_requires` in `setup.py`.

### PR DESCRIPTION
**Pull request by ziima**

### CODE REVIEW

File: tests/test_appsettings.py [hunk 1]
```diff
@@ -25,15 +25,15 @@ def setUp(self):
         self.message_no_attr = "has no attribute '%s'"
 
     def test_setting(self):
-        setting = appsettings.Setting()
-        assert setting.name == ""
-        assert setting.full_name == ""
+        setting = appsettings.Setting(name='simple')
+        assert setting.name == 'simple'
+        assert setting.full_name == 'SIMPLE'
```
Comment (pawamoy) [inline tests/test_appsettings.py]: Cannot settings be empty anymore?

File: setup.py [hunk 2]
```diff
@@ -68,6 +69,7 @@ def read(*names, **kwargs):
     keywords=[
         'django', 'app', 'settings',
     ],
+    install_requires=['Django'],
```
[the remark this answers is not in the document]
Comment (ziima) [reply setup.py]: I think so. The appsettings really need Django to run. I don't really see any benefit of not declaring it as a dependency.

File: tests/test_appsettings.py (hunk 1 above)
Comment (ziima) [reply tests/test_appsettings.py]: The `AttributeError` now contains only the name of the missing attribute. With empty name of the settings, the `AttributeError` contains only an empty string. That can't be properly checked in test below.

File: setup.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (pawamoy) [reply setup.py]: Fair enough!

**Pull request merged**